elasticsearch/plan.sh

```diff
@@ -38,7 +38,7 @@ do_install() {
   cp -a ./* "${pkg_prefix}/es"
 
   # jvm.options needs to live relative to the binary.
-  # mkdir -p mkdir -p "$pkg_prefix/es/config"
+  # mkdir -p "$pkg_prefix/es/config"
```

Comment:
wait, any reason these are still commented out?

Reply:
Nope. It didn't block previous merges so I think we should move ahead :)

Reply:
ack, sorry, I thought I approved with comment D:  proceed!